packages/SwingSet/test/upgrade/test-upgrade.js

```diff
@@ -74,11 +74,7 @@ async function testUpgrade(t, defaultManagerType) {
   const hostStorage = provideHostStorage();
   const { kvStore } = hostStorage;
   await initializeSwingset(config, [], hostStorage);
-  const c = await makeSwingsetController(
-    hostStorage,
```

Comment:
assuming this is misc cleanup

Reply:
Yes.  Debugging debris, presumably left over from @warner working on upgrade tests.